includes/wikia/tasks/Tasks/MultiTask.class.php

```diff
@@ -159,38 +165,92 @@ public function edit( $params ) {
 			$commandParams
 		);
 
-		return array_sum($result);
+		return array_sum( $result );
 	}
 
-	private function parseCommon($params) {
-		$impersonatedName = isset($params['user']) ? $params['user'] : 'Maintenance script';
-		$createdBy = \User::newFromId($this->createdBy);
-		$impersonatedUser = \User::newFromName($impersonatedName);
-		$page = isset($params['page']) ? \Title::newFromText($params['page']) : null;
+	private function parseCommon( $params ) {
+		$impersonatedName = isset( $params['user'] ) ? $params['user'] : 'Maintenance script';
+		$createdBy = \User::newFromId( $this->createdBy );
+		$impersonatedUser = \User::newFromName( $impersonatedName );
+		$page = isset( $params['page'] ) ? \Title::newFromText( $params['page'] ) : null;
 
-		if (!is_object($impersonatedUser)) {
+		if ( !is_object( $impersonatedUser ) ) {
 			$impersonatedName = '';
 		}
 
 		return [$createdBy, $impersonatedName, $page];
 	}
-	
-	private function runOnWikis($wikiInputRaw, $lang, $cat, $wikiId, $action, $commandParams) {
+
+	/**
+	 * Get the list of wikis where the given article exists
+	 *
+	 * Run this query to get the list of wikis where MultiTask should be run
+	 * instead of executing the task on all active wikis
+	 *
+	 * Example: SELECT * FROM pages WHERE page_title_lower = 'tomyvilu' AND page_namespace = 2;
+	 *
+	 * @see PLATFORM-783
+	 *
+	 * @param \Title $title
+	 * @return array
+	 */
+	private function getWikisWherePageExists( \Title $title ) {
+		global $wgExternalDatawareDB;
+
+		$dbr = wfGetDB( DB_SLAVE, [], $wgExternalDatawareDB );
+		$res = $dbr->select(
+			'pages',
+			'page_wikia_id as wiki_id',
+			[
+				'page_title_lower' => str_replace( ' ', '_', mb_strtolower( $title->getText() ) ),
```

Comment:
Any reason not to just use `->getDBkey()` instead doing the replace on `getText()`?